src/pytezos/context/abstract.py

```diff
@@ -224,3 +228,6 @@ def set_voting_power(self, address: str, voting_power: int):
 
     def get_operations_ttl(self) -> int:
         raise NotImplementedError
+
+    def get_min_block_time(self) -> int:
+        raise NotImplementedError
```

Comment:
Code bugs in the above! Why can't people use exceptions correctly?!

Anyway - no need to change as this code never runs!